_kits/it/web-analytics.md

```diff
@@ -40,24 +40,31 @@ footerItems:
 superAccordion:
   title: ATTIVITÀ E RISORSE
   accordions:
-    - title: Usa Web Analytics Italia
-      text: "<p>La piattaforma Web Analytics Italia (WAI) è una soluzione open source creata ad hoc dall'Agenzia per l'Italia Digitale per la raccolta e l'analisi dei dati di traffico dei siti web delle amministrazioni pubbliche.</p> <p>WAI ha la finalità di aiutare gli operatori della PA a migliorare l'usabilità dei <em>touchpoint</em> digitali grazie al monitoraggio continuo, l'elaborazione e la condivisione dei dati statistici di analisi web dei siti web aderenti al progetto, in completa aderenza alla normativa GDPR.</p>"
+    - title: Monitora i dati di traffico della tua soluzione digitale
+      text: "<p>Esistono numerosi software open source per la raccolta e l'analisi dei dati di traffico di siti e servizi digitali. In questa scheda ti proponiamo una lista di alcune fra le principali piattaforme che puoi utilizzare per il tracciamento, l'elaborazione e la condivisione dei dati statistici della tua soluzione digitale, in completa aderenza alla normativa GDPR.</p>"
       img: "/assets/icons/kit/kit_analytics_usa.svg"
       resources:
-        - title: La piattaforma Web Analytics Italia
+        - title: Matomo analytics
           icon: "/assets/icons/pencil.svg"
-          description: Un servizio open source dedicato alla raccolta e all'analisi dei dati di traffico dei servizi digitali della PA
-          link: <a href='https://webanalytics.italia.it/' aria-label="Vai alla piattaforma WAI (link esterno)">Vai alla piattaforma</a>
+          description: "Matomo è una piattaforma di web analytics open source conforme alla normativa privacy in vigore."
+          link: <a href='https://developers.italia.it/it/software/italia-software-matomo-32c75d' aria-label="Vai alla scheda di Matomo analytics (link esterno)">Vai alla scheda di Matomo analytics</a>
```

Comment:
Per il link esterno non avete un widget particolare in BSI?

Reply:
non saprei sai, @Fupete ?

Reply:
Non ho capito. Il link <a qui è corretto data la struttura dell'attuale Designers. Nel next in React avremo un automatismo per questa cosa.

Reply:
Scusa, c'è già scritto sotto ho visto solo ora `Tutte le risorse sono collegamenti esterni che si aprono in una nuova finestra.` Quindi direi che va bene così com'è ora, grazie!